src/main/scala/diplomaticobjectmodel/model/OMISA.scala

```diff
@@ -82,10 +82,11 @@ object OMISA {
     }
 
     val addressTranslationModes = xLen match {
-        case _ if !coreParams.useVM => Bare
-        case 32 => Sv32
-        case 64 => Sv39
-        case _ => throw new IllegalArgumentException(s"ERROR: Invalid Xlen: $xLen")
+      case _ if !coreParams.useVM => Bare
```

Comment:
confused why this change is in this PR

Reply:
yeah its because I needed 3.3 stuff. I'll make the PR against chisel-3-3